### PR TITLE
Adds public event CFS component (UI only)

### LINKED ISSUE #54: Add public event Call for Speakers page (UI Only)
screen shot 2017-05-16 at 6 40 41 pm

- Located at `/events/<event_identifier/cfs` - route already exisits
- Should be a separate call for speakers component.
- Should support two states. (Call for speakers open and closed)
- Dummy data to be used
- The submit proposal button need not do anything

### PR DESCRIPTION
**Pull request by CosmicCoder96**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Adds the public call for speaker component (UI Only)
#### Changes proposed in this pull request:
image

image

Fixes #54

### CODE REVIEW

**Comment by CosmicCoder96** [comment]
@niranjan94 Please review.

File: app/templates/components/public/call-for-speakers.hbs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<h1 class="ui header">
+  Call for Speakers
+  {{#if call.isClosed}}
+    <a class="ui basic red label">Closed</a>
+    {{t Call closed}}
```
Comment (niranjan94) [inline app/templates/components/public/call-for-speakers.hbs]: Not required.

File: app/templates/components/public/call-for-speakers.hbs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  {{else}}
+    <a class="ui basic green label">Open </a>
+    <div class="sub header">
+      {{t 'Call Open until'}} Fri, Oct 13 2017 5:00 pm
```
Comment (niranjan94) [inline app/templates/components/public/call-for-speakers.hbs]: Let the text be `Call for speakers open until`

File: app/templates/components/public/call-for-speakers.hbs [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<h1 class="ui header">
+  Call for Speakers
+  {{#if call.isClosed}}
+    <a class="ui basic red label">Closed</a>
```
Comment (niranjan94) [inline app/templates/components/public/call-for-speakers.hbs]: Put closed inside translation helper

File: app/templates/components/public/call-for-speakers.hbs [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    {{t Call closed}}
+  {{else}}
+    <a class="ui basic green label">Open </a>
```
Comment (niranjan94) [inline app/templates/components/public/call-for-speakers.hbs]: Put open inside translation helper

File: app/templates/components/public/call-for-speakers.hbs [hunk 5]
```diff
@@ -0,0 +1,36 @@
+<h1 class="ui header">
+  Call for Speakers
+  {{#if call.isClosed}}
```
Comment (niranjan94) [inline app/templates/components/public/call-for-speakers.hbs]: Instead of `call` .. name the variable as `callForSpeakers`

**Comment by geekyd** [comment]
@CosmicCoder96 Great work!
I see the text is not using the translation helper, can you please add it to the text in the HTML?

**Comment by CosmicCoder96** [comment]
@niranjan94 I have made the changes. Please review again.

**Pull request merged**